.travis.yml

```diff
@@ -24,3 +24,5 @@ after_success:
   - bin/createcoverage
   - pip install coverage==3.7.1 coveralls
   - coveralls
+  - docker.io build --tag plone.restapi:$(git rev-parse HEAD) .
+#  - docker.io push kitconcept/plone.restapi
```

Comment:
@svx I don't want to run docker on travis. Just automatically build the latest version and push it to dockerhub. Other projects should use the plone.restapi docker service on travis then in order to run tests against it (with their custom front-ends for instance).